app/src/main/java/de/tum/in/tumcampusapp/api/app/TUMCabeClient.java

```diff
@@ -366,23 +366,34 @@ public void getChatMemberByLrzId(String lrzId, Callback<ChatMember> callback) {
 
     // Getting event information
     public void fetchEvents(Callback<List<Event>> cb) {
-        service.getEvents().enqueue(cb);
+        // TODO remove
```

Comment:
agreed.
Still wanna do this in this PR?

Reply:
Forgot that 🙈 – removed now